SR2020/2020-05-26-virtual-competition-details.md

```diff
@@ -0,0 +1,32 @@
+---
+to: sr2020-teams
+subject: Student Robotics 2020 Virtual Competition Details
+---
+
+![](https://pbs.twimg.com/media/EYt9wgcX0AAKjOn)
+
+Hello!
+
+Today's the day we announce some awaited details of the virtual competition for Student Robotics 2020.
+
+## Simulator
+
+For the virtual competition, we've created a new simulator, based on [webots](https://cyberbotics.com/). Documentation on how to download and get started with the simulator is available on the docs.
+
+[Competition Simulator Docs](https://studentrobotics.org/docs/competition-simulator/)
+
+Due to some compatibility and API changes, it's unlikely your existing code will work exactly in the simulator.
```

Comment:
Maybe needs a "but we have tried to keep it similar enough that you can easily adapt what you've already written" or something along those lines?